style.css

```diff
@@ -98,3 +103,241 @@ a {
 p a {
   color: #7f8cff;
 }
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 2em;
+  padding: 100px 25px;
+}
+
+.card {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 20px;
+  min-height: 80vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.pic1 img {
+  width: 100%;
+}
+
+.title1 {
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+}
+
+.frame-container {
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  margin-top: -12px;
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  padding: 0%;
+  margin-left: -5px;
+}
+
+.role,
+.year {
+  color: #7a869a;
+}
+
+.card-text {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
```

Comment:
- [ ] Please make sure to use the given color for this. In Figma template, it is `#344563` because of the above mentioned reasons 👆
- [ ] Also, could you please reduce the space/margin between the following mentioned area for getting a design close to the Figma template as much as possibe? 🔨🤓

Your design | Given design 
---| ---|
![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/094ed393-4ee0-467f-aeb3-cd6cc944df4d) | ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/62bf19d0-fd50-42c9-92ec-f253e63bfc6b)



_Kindly apply this to all project cards_